spec/spec_helper.rb

```diff
@@ -69,6 +69,22 @@
     DatabaseCleaner.strategy = :deletion
   end
 
+  config.before(:each, stub_helper: true) do
+    @eq_model1 = stubbed_eq_model
+    @eq_model2 = stubbed_eq_model
+    @eq_model3 = stubbed_eq_model
+    @helper = OrderingHelper.new(@eq_model2)
+    allow(@helper).to receive(:ordering).and_return(3)
+    allow(@helper).to receive(:predecessor).and_return(@eq_model1)
+    allow(@helper).to receive(:successor).and_return(@eq_model3)
+    allow(@helper).to receive(:rel_predecessor).with(3).and_return(1)
+    allow(@helper).to receive(:rel_successor).with(3).and_return(5)
+    allow(@helper).to receive(:cat_first).and_return(1)
+    allow(@helper).to receive(:cat_last).and_return(5)
+    allow(@helper).to receive(:successors).and_return([@eq_model3])
+    allow(@helper).to receive(:rel_predecessor).with(5).and_return(3)
+  end
+
```

Comment:
This is the stub_ordering_helper method pulled out into a before block. Which approach is better (or what is the correct approach, if neither)?